apps/website/src/components/mdx-components/index.tsx

```diff
@@ -10,12 +10,6 @@ import { KeyboardInteractionTable } from '../keyboard-interaction-table/keyboard
 import { Note } from '../note/note';
 import { Showcase } from '../showcase/showcase';
 import { StatusBanner } from '../status-banner/status-banner';
-import {
-  Accordion,
-  AccordionContent,
-  AccordionItem,
-  AccordionTrigger,
-} from '@qwik-ui/styled';
```

Comment:
shouldn't we keep this import? some .mdx files are probably using the styled accordion. I would simply replace Accordion by AccordionRoot since you renamed it like that.